packages/react-adminer/src/components/Modal.tsx

```diff
@@ -0,0 +1,39 @@
+import { Modal as ModalAntd } from 'antd';
+
+interface Props {
+	title: string;
+	setIsModalVisible: React.Dispatch<React.SetStateAction<boolean>>;
+	isModalVisible: boolean;
+	okText: string;
+	content: any;
+	onOk: any;
+}
+
+const Modal: React.FC<Props> = ({ title, setIsModalVisible, isModalVisible, okText, content, onOk }) => {
+	const handleOk = (): void => {
+		setIsModalVisible(false);
+	};
+	const handleCancel = (): void => {
+		setIsModalVisible(false);
+	};
+
+	return (
+		<>
+			<ModalAntd
+				title={title}
+				visible={isModalVisible}
+				onOk={() => {
+					handleOk();
```

Comment:
ty funkce delaj to stejne, je to skoda ne? vola se to dvakrat a dvakrat to udela neco v reactu